doc/orchestrator.rst

```diff
@@ -1,207 +1,725 @@
+  using simple put/get semantics in SmartRedis.
+
+In scenarios with high data throughput, such as online analysis, training, and processing, a standalone ``Orchestrator``
+is optimal. The data produced by multiple processes in a SmartSim ``Model`` is stored in the standalone
+``Orchestrator`` and is available for consumption by other SmartSim ``Models``.
+
+If a workflow requires an application to leverage multiple standalone deployments,
+multiple ``Clients`` can be instantiated within an application,
+with each ``Client`` connected to a unique deployment. This is accomplished through the use of the
+`db-identifier` and :ref:`ConfigOptions<config_options_explain>` object specified at ``Orchestrator`` initialization time.
+
+-------
+Example
+-------
+In the following example, we demonstrate deploying a standalone ``Orchestrator`` on an HPC System.
+Once the standalone ``Orchestrator`` is launched from the driver script, we walk through
+connecting a SmartRedis ``Client`` to the ``Orchestrator`` from within the SmartSim ``Model``
+script to transmit and poll for data.
 
+The example is comprised of two script files:
 
-Cluster Orchestrator
-====================
+- :ref:`Application Script<standalone_orch_app_script>`
+   The application script is a Python file that contains instructions to create a SmartRedis
+   ``Client`` connection to the standalone ``Orchestrator`` launched in the driver script.
+   To demonstrate the ability of workflow components to access data from
+   other entities, we then retrieve the tensors set by the driver script using a SmartRedis ``Client`` in
+   the application script. We then instruct the ``Client`` to send and retrieve data from within the application script.
+- :ref:`Experiment Driver Script<standalone_orch_driver_script>`
+   The ``Experiment`` driver script is responsible for launching and managing SmartSim entities. Within this script,
+   we use the ``Experiment`` API to create and launch a standalone ``Orchestrator``. To demonstrate the capability of
+   SmartSim ``Model`` applications to access ``Orchestrator`` data sent from other sources, we employ the SmartRedis ``Client`` in
+   the driver script to store a tensor in the ``Orchestrator``, which is later retrieved by the SmartSim ``Model``.
+   To employ the application script, we initialize a ``Model`` object with the application script as an executable argument,
+   launch the ``Orchestrator``, and then launch the ``Model``.
+
+   To further demonstrate the ability of workflow components to access data from
+   other entities, we then retrieve the tensors stored by the ``Model`` using a SmartRedis ``Client`` in
+   the driver script. Lastly, we tear down the ``Orchestrator``.
+
+.. _standalone_orch_app_script:
+Application Script
+==================
+To begin writing the application script, import the necessary SmartRedis packages:
+
+.. code-block:: python
+
+  from smartredis import Client, log_data
+  from smartredis import *
```

Comment:
Do we need both the   from smartredis import Client, log_data and   from smartredis import * ?  It seems like the latter would take care of everything but we probably want to avoid * imports.  Thoughts?

Reply:
let me run this real quick and check